core/network/src/chain.rs

```diff
@@ -80,10 +81,10 @@ impl<B, E, Block, RA> Client<Block> for SubstrateClient<B, E, Block, RA> where
 	Block: BlockT<Hash=H256>,
 	RA: Send + Sync
 {
-	fn import(&self, block: ImportBlock<Block>, new_authorities: Option<Vec<AuthorityIdFor<Block>>>)
+	fn import(&self, block: ImportBlock<Block>, new_cache: HashMap<Vec<u8>, Vec<u8>>)
```

Comment:
Not related to this PR, but could you, please, remove this method (from both trait and this impl) - it is obsoleted by moving `ImportQueue` to the separate crate.